### DISCUSSION TITLE
How to pass Vector of priors?

### DISCUSSION THREAD

**Discussion by draconcio12** [pymc-devs/pymc3#6856] [Q&A]
How I can pass a vector of priors?

I try using shape of n_var using this code,  but I guet this error:

 ""Cannot broadcast np.ndarray with operand of type <class 'pytensor.tensor.var.TensorVariable'>" "

with pm.Model() as model:

    params= pm.Normal('params', mu=prior_vector, shape=n_var)

    likelihood= pm.Normal('y', mu= funcion(params),  observed=y)

    trace = pm.sample(1000)

**Comment by fonnesbeck** [comment]
Its not exactly clear what you are trying to do. How does `n_var` relate to the size of `y`. Also, what is the function you are calling on `params`? 

It would be helpful to have a runnable example.

**Comment by twiecki** [comment]
These type of questions should be directed to https://discourse.pymc.io/. These discussions are for dev-related topics.
